.github/workflows/ci.yml

```diff
@@ -24,8 +24,8 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - name: Ktlint
-        run: ./gradlew ktlint
+      - name: Detekt # style formatter and static analysis
+        run: ./gradlew paymentsheet-example:detekt stripe:detekt stripe-test-e2e:detekt
```

Comment:
Can it be run on the top-level project?

Reply:
It didn't work when I first tried it, but now it works!